rasterio/plot.py

```diff
@@ -327,9 +334,25 @@ def show_hist(
         plt.show()
 
 
+def hist_strech(arr, clip_percent):
```

Comment:
@xinluo2018 will you change this to `contrast_stretch(arr, percent_range=(2.0, 98.0))`? That would make the API more like `skimage.exposure.rescale_intensity()`, which would be nice.

Reply:
we have change the script by following you suggestion.